src/FileIndexer.ts

```diff
@@ -378,7 +395,11 @@ export class FileIndexer {
       return this.cached(node, this.scipSymbol(node.parent))
     }
 
-    if (ts.isImportSpecifier(node) || ts.isImportClause(node)) {
+    if (
+      ts.isImportSpecifier(node) ||
+      ts.isImportClause(node) ||
+      ts.isNamespaceImport(node)
```

Comment:
Is this change related to de-duplication? It would be nice to isolate this change and have a diff in the snapshot tests to better understand the exact motivation for this change.

Reply:
https://github.com/sourcegraph/scip-typescript/pull/195